tgui/packages/tgui/components/ByondUi.js

```diff
@@ -78,19 +72,24 @@ export class ByondUi extends Component {
     this.handleResize = debounce(() => {
       this.forceUpdate();
     }, 100);
+
+    let lock = false;
+    this.handleScroll = () => {
+      if (!lock) {
+        window.requestAnimationFrame(() => {
+          this.componentDidUpdate();
```

Comment:
What was the problem?

Reply:
`onscroll` handler was never cleaned up in `componentWillUnmount` and since the onscroll behavior is happening on a div rendered by `<Window.Content scrollable>` and the event never bubbles up, the event was never fired, thus required the `capture` option on the `addEventListener`.

Reply:
I feel like there was a discussion in #imcoder about this...